### DISCUSSION TITLE
Admin UI baseHref

### DISCUSSION THREAD

**Discussion by BenVuti** [vendurehq/vendure#992] [Q&A]
When trying to locally setup the [compiling-as-a-deployment-step](https://www.vendure.io/docs/plugins/extending-the-admin-ui/#compiling-as-a-deployment-step) I ran into the issue of the /admin page being blank.

I tried modifying the `<base href="/" />` to `<base href="/admin" />` in the index.html file but it didn't fix the problem.

I then deleted the entire admin-ui folder, added the following line to the `compileUiExtensions`
```
compileUiExtensions({
    outputPath: path.join(__dirname, 'admin-ui'),
    baseHref: '/admin', // <----
    extensions: [/* ... */],
}).compile?.().then(() => {
    process.exit(0);
});
```
I rebuilt and tried to access the admin again, but the page remained blank.
By inspecting the source code of the page I can see that the baseHref is still "/"

**Comment by michaelbromley** [comment]
I converted this into an issue here: https://github.com/vendure-ecommerce/vendure/issues/993
